tests/components/timer/test_init.py

```diff
@@ -320,31 +320,55 @@ async def test_start_service(hass: HomeAssistant) -> None:
     assert state.attributes[ATTR_DURATION] == "0:00:15"
     assert state.attributes[ATTR_REMAINING] == "0:00:15"
 
+    with pytest.raises(
+        HomeAssistantError,
+        match="Not possible to change timer timer.test1 beyond configured duration",
+    ):
+        await hass.services.async_call(
+            DOMAIN,
+            SERVICE_CHANGE,
+            {CONF_ENTITY_ID: "timer.test1", CONF_DURATION: 20},
+            blocking=True,
+        )
+        await hass.async_block_till_done()
```

Comment:
The service call is already marked as blocking, there is no need to block again.

```suggestion
```